manifests/plugin/write_prometheus.pp

```diff
@@ -1,5 +1,6 @@
 class collectd::plugin::write_prometheus (
   Stdlib::Port $port = 9103,
+  Optional[Stdlib::IP::Address] $ip = undef,
```

Comment:
please document the new parameter in the class with puppet-string style:
```
# @param ip this does blabla
```

Reply:
Sure will do that Tue or so